src/pybind/mgr/dashboard/frontend/src/app/shared/services/summary.service.ts

```diff
@@ -40,7 +42,46 @@ export class SummaryService {
     });
   }
 
+  /**
+   * Returns the current value of summaryData
+   *
+   * @returns
+   * @memberof SummaryService
+   */
   get() {
-    return this.http.get('api/summary');
+    return this.summaryDataSource.getValue();
+  }
+
+  /**
+   * Subscribes to the summaryData,
+   * which is updated at least once every 5 seconds
+   *
+   * @param {(summary: any) => void} call
+   * @returns {Subscription}
+   * @memberof SummaryService
+   */
+  subscribe(call: (summary: any) => void): Subscription {
+    return this.summaryData$.subscribe(call);
+  }
+
+  /**
+   *
+   *
+   * @param {ExecutingTask} task
+   * @memberof SummaryService
```

Comment:
This comment doesn't bring any value, please remove it.

Reply:
I have updated the comment.